homeworks/olya.kosenko_olyabot/homework_2/Task_1.js

```diff
@@ -0,0 +1,30 @@
+function pyramid(n1) {
+    console.log(`Pyramid with ${n1} rows:`);
+    const str1 = '#';
+    const str2 = ' ';
+    for (let i = 0; i < n1; i++) {
+        console.log(str2.repeat(n1 - i) + str1.repeat(i * 2 + 4));
+    }
+}
+
+function check(n1) {
+    return (!Number.isNaN(n1) && n1 > 0) ? 1 : 0;
```

Comment:
:) Ok, I will merge it, but it's better to simplify to:
`    return !Number.isNaN(n1) && n1 > 0;`